lib/thrift/generator.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Thrift.Generator do
   alias Thrift.Parser.FileGroup
   alias Thrift.Generator.EnumGenerator
+  alias Thrift.Generator.Service, as: ServiceGenerator
```

Comment:
Then shouldn't you name your new file `generator/service.ex` instead of `generator/service_generator.ex`?